tools/getkeys/index.js

```diff
@@ -36,14 +39,15 @@ async function exportToShellRc(shellRc, entries) {
 // Persists the given 'env' map to the users environment.  The method used depends
 // on the user's platform and login shell.
 async function saveEnv(env) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
Rather than disabling all of the typescript rules inline here. It might be better to disable them in the eslint config file. This is all javascript code, so none of these rules should really be being applied at all (but our base config clearly has some problems we need to fix).

It might be worth moving them to the eslint config file with a comment that we should remove all of the ts-specific rule exceptions when the base config is configured better for javascript.